pwiz_tools/Skyline/Controls/Graphs/DetectionToolbarProperties.Designer.cs

```diff
@@ -28,6 +28,7 @@ protected override void Dispose(bool disposing)
         /// </summary>
         private void InitializeComponent()
         {
+            System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(DetectionToolbarProperties));

```

Comment:
Seems like this should conflict with changes I made during merging of the 20.2 RESX file translation

Reply:
Done